src/coprocessor/dag/rpn_expr/impl_arithmetic.rs

```diff
@@ -0,0 +1,334 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
+
+use cop_codegen::RpnFunction;
+use cop_datatype::FieldTypeAccessor;
+
+use super::types::RpnFnCallPayload;
+use crate::coprocessor::codec::data_type::*;
+use crate::coprocessor::codec::mysql::{Decimal, Res};
+use crate::coprocessor::codec::Error;
+use crate::coprocessor::dag::expr::EvalContext;
+use crate::coprocessor::Result;
+use cop_datatype::FieldTypeFlag;
+
+#[derive(Debug, Clone, Copy, RpnFunction)]
+#[rpn_function(args = 2)]
+pub struct RpnFnIntMod;
+
+impl RpnFnIntMod {
```

Comment:
You need to separate it into different functions, so that no SIGNED/UNSIGNED check inside. Take a look at GTInt as examples.

Reply:
What about this way? https://github.com/sticnarf/tikv/commit/7df0db543b3bf164fb2204536aec9ed20156fc14

Reply:
@sticnarf OK, I will address it after your PR merged.